web/src/components/lineage/components/node/Node.tsx

```diff
@@ -43,25 +46,52 @@ const Node: React.FC<NodeProps> = ({ node, selectedNode, setSelectedNode }) => {
     return '/'
   }
 
+  // configure the tool tip
+  const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
+    <Tooltip {...props} classes={{ popper: className }} />
+  ))(({ theme }) => ({
+    [`& .${tooltipClasses.tooltip}`]: {
+      backgroundColor: theme.palette.background.default,
+      color: theme.palette.common.white,
+      maxWidth: 600,
+      fontSize: theme.typography.pxToRem(14),
+      border: '1px solid ' + theme.palette.common.white,
+    },
```

Comment:
It would be our preference moving forward to avoid this type of css override and instead use the default properties as much as possible since there are now multiple maintainers it's becoming a little challenging to find consensus here. I would try to only use the properties available to you here without stylesheet overrides.